src/raftstore/store/peer.rs

```diff
@@ -401,7 +425,8 @@ impl Peer {
         if req.has_admin_request() {
             match req.get_admin_request().get_cmd_type() {
                 AdminCmdType::CompactLog |
-                AdminCmdType::InvalidAdmin => {}
+                AdminCmdType::InvalidAdmin |
+                AdminCmdType::TransferLeader => {}
```

Comment:
There are two place called `check_epoch`, one is `exec_raft_cmd` which won't be executed. the other is `propose`. 
Check epoch in propose maybe ok through, I haven't thought out any serious problem if we don't.
Suppose conf changed, transfer to another leader won't make much trouble, although it's not right behavior, TransferLeader itself is a command never guaranteed success. Transfer to a non-exist peer can't success in underlying raft. The same to region split case.
Btw, will we get a stale TransferLeader command?